crates/api_common/src/utils.rs

```diff
@@ -516,6 +518,59 @@ pub fn local_site_opt_to_sensitive(local_site: &Option<LocalSite>) -> bool {
     .unwrap_or(false)
 }
 
+async fn get_url_blocklist() -> &'static RwLock<Option<RegexSet>> {
+  static URL_BLOCKLIST: Lazy<RwLock<Option<RegexSet>>> = Lazy::new(|| RwLock::new(None));
```

Comment:
This doesn't seem to be updated when another lemmy server with the same database updates the blocklist. It should use something similar to the [federation blocklist cache](https://github.com/LemmyNet/lemmy/blob/10bf7464b1b22227ae3fa246a9cd6327a2617aaf/crates/apub/src/lib.rs#L122).

`update_url_blocklist` can still work using `CACHE.invalidate_all()`.